app/routes/object_storage/objects.py

```diff
@@ -61,7 +61,15 @@ def put_object(namespace_name, bucket_name, subpath):
         }
     )
 
-    return ""
+    return Response(
+        status=200,
+        headers={
+            "etag": str(uuid.uuid4()),
```

Comment:
This way the etag of an object will change each time you look for it.
I think it would make more sense if you add this information to the inserted object and then returning it here so the etag won't change.

```python
    bucket["_objects"].append(
        {
            "cache_control": cache_control,
            "content_type": content_type,
            "content_type": content_type,
            "object_name": subpath,
            "ref_obj": ref_obj,
            "content_disposition": content_disposition,
            "etag": str(uuid.uuid4()),
        }
    )
    
    return Response(
        status=200,
        headers={
            "etag": bucket["_objects"][-1]["etag"],
            "opc-request-id": request.headers["Opc-Request-Id"]
            if "Opc-Request-Id" in request.headers
            else "",
        }
    )
  ```